Sources/EdgeNetworkHandlers/EdgeEndpoint.swift

```diff
@@ -13,7 +13,7 @@
 import Foundation
 
 /// Represents all the known endpoints for the Edge Network
```

Comment:
Let's update the comments on this enum since is not directly related to the endpoints. E.g. Represents all the known Edge Network environment types